src/TestKernel.php

```diff
@@ -181,10 +188,14 @@ protected function configureRoutes($routes): void
         }
     }
 
-    public function shutdown(): void
+    public function shutdown($clearCache = true): void
```

Comment:
As far as I can see, this isn't used anymore:

```suggestion
    public function shutdown(): void
```

Reply:
Oops. Thank you.